### PR TITLE
Header

### PR DESCRIPTION
**Pull request by patofernandez**
image

### CODE REVIEW

File: wbooks/src/app/components/Header/components/BackButton/styles.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from 'react-native';
+import { white, cyan } from '../../../../../constants/colors';
```
Comment (mattgle) [inline wbooks/src/app/components/Header/components/BackButton/styles.js]: add space between imports

File: wbooks/src/app/components/Header/components/SearchButton/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+import { 
+	Image,
+	TouchableOpacity
+} from 'react-native';
+
+import styles from './styles';
+import searchImage from '../../../../../assets/ic_search.png';
```
Comment (mattgle) [inline wbooks/src/app/components/Header/components/SearchButton/index.js]: same

File: wbooks/src/app/components/Header/components/SearchButton/styles.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { StyleSheet } from 'react-native';
+import { white, cyan } from '../../../../../constants/colors';
```
Comment (mattgle) [inline wbooks/src/app/components/Header/components/SearchButton/styles.js]: same

File: wbooks/src/app/components/Header/index.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+import { 
+  View,
+  Text,
+	Image,
+	TouchableOpacity
+} from 'react-native';
+
+import backgroundImage from '../../../assets/bc_nav_bar.png';
+import backImage from '../../../assets/ic_back.png';
+import searchImage from '../../../assets/ic_search.png';
```
Comment (mattgle) [inline wbooks/src/app/components/Header/index.js]: add space, cmon u know how to

File: wbooks/src/app/components/Header/index.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+import SearchButton from './components/SearchButton';
+import styles from './styles';
+
+class CustomHeader extends Component {
+	render() {
+		return (
+			<View style={styles.container}>
+				<Image style={styles.image}
+					source={backgroundImage}
+					resizeMode="stretch"
```
Comment (mattgle) [inline wbooks/src/app/components/Header/index.js]: U sure is not contain?

File: wbooks/src/app/components/Header/styles.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
+import { StyleSheet } from 'react-native';
+import { white, cyan } from '../../../constants/colors';
```
Comment (mattgle) [inline wbooks/src/app/components/Header/styles.js]: add space between imports

File: wbooks/src/app/components/Header/styles.js [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+const imageButtonSize = 20;
+
+export default StyleSheet.create({
+  container: {
+    height: 102,
+    alignContent: 'center',
+    backgroundColor: cyan
+  },
+  image: {
+    position: 'absolute',
```
Comment (mattgle) [inline wbooks/src/app/components/Header/styles.js]: Why absolute?

Comment (mattgle) [reply wbooks/src/app/components/Header/styles.js]: Come to my desk later and we'll talk this

File: wbooks/src/app/components/Header/components/SearchButton/index.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+import { Image, TouchableOpacity } from 'react-native';
+
+import searchImage from '../../../../../assets/ic_search.png';
+
+import styles from './styles';
+
+class SearchButton extends Component {
+	render() {
+		return (
+      <TouchableOpacity>
```
Comment (guidoprinc) [inline wbooks/src/app/components/Header/components/SearchButton/index.js]: this should have an onPress function prop

File: wbooks/src/app/components/Header/index.js [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+
+import backgroundImage from '../../../assets/bc_nav_bar.png';
+import backImage from '../../../assets/ic_back.png';
+import searchImage from '../../../assets/ic_search.png';
+
+import BackButton from './components/BackButton';
+import SearchButton from './components/SearchButton';
+import styles from './styles';
+
+class CustomHeader extends Component {
```
Comment (guidoprinc) [inline wbooks/src/app/components/Header/index.js]: arrow function since you are using only a render

**Pull request merged**